src/scss/components/button/_mixins.scss

```diff
@@ -13,10 +13,14 @@
   padding: 0;
 }
 
-@mixin button-basic {
+@mixin button-base {
   border-radius: 2px;
   display: inline-flex;
   justify-content: center;
+}
+
+@mixin button-common { // less: #Button.basic()
+  @include button-base;
```

Comment:
The `border`, `display`, and `justify-content` properties were split into a new `button-base` mixin so that I didn't have to pull in more styles than necessary into Shadow DOM CSS (keeping it as small as possible).

Reply:
Changes to this file will be handled in #587.